sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -1188,25 +1398,16 @@ func processConnection(conn *sensor.NetworkConnection) (*connection, error) {
 	return c, nil
 }
 
-func getUpdatedConnections(hostname string, networkInfo *sensor.NetworkConnectionInfo) map[connection]timestamp.MicroTS {
+func getUpdatedConnections(_ string, networkInfo *sensor.NetworkConnectionInfo) map[connection]timestamp.MicroTS {
```

Comment:
```suggestion
func getUpdatedConnections(networkInfo *sensor.NetworkConnectionInfo) map[connection]timestamp.MicroTS {
```

Reply:
Same for `getUpdatedContainerEndpoints`

Reply:
done in [1ef1615](https://github.com/stackrox/stackrox/pull/14538/commits/1ef16157ac6eff064f1af8e8da85d1e47c8348bc)